pkg/server/models.go

```diff
@@ -0,0 +1,17 @@
+package server
+
```

Comment:
How do we use these maps?

Reply:
The getRecommendations request from the user will have either a RecommendationID (ex: `ns_local-cluster_open-cluster-management_123432112`) or the ClusterNamespace (ex `local-cluster` and `open-cluster-management`). Depending what is given we use either ClusterNamespaceMap or RecommendationIDMap to get to the RecommendationStatusMap which will hold the recommendation data we need to make request to kruize and store the recommendation status.